styles/components/quests/quiz.module.css

```diff
@@ -403,16 +418,19 @@
     width: calc(100% - 70px) !important;
     flex-direction: column;
     align-items: baseline;
+    margin-top: 25px;
     justify-content: baseline;
   }
 
   .checkBoxContainer {
     width: 100%;
-    margin: 25px 0 !important;
+    /* margin: 25px 0 !important; */
   }
 
   .tableLayout {
     width: 100%;
+   grid-template-columns: 100%;
+   /* gap: 0; */
```

Comment:
same here

Reply:
okk sure i will remove the commented code